README.md

```diff
@@ -1,25 +1,30 @@
-[![Godot Engine logo](/logo.png)](https://godotengine.org)
+# Godot Engine
 
-## Godot Engine
+<p align="center">
+  <a href="https://godotengine.org">
+    <img src="logo.svg" width="400" alt="Godot Engine logo">
+  </a>
+</p>
 
-Homepage: https://godotengine.org
+## 2D and 3D cross-platform game engine
 
-#### 2D and 3D cross-platform game engine
+**[Godot Engine](https://godotengine.org) is a feature-packed, cross-platform
+game engine that can be used to create 2D and 3D games from a unified
+interface.** It provides a comprehensive set of common tools, so that users can
+focus on making games without having to reinvent the wheel. Games can be
+exported in one click to a number of platforms, including the major desktop
+platforms (Linux, macOS, Windows), mobile platforms (Android, iOS), as well as
+Web-based platforms (HTML5) and
+[consoles](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html)
```

Comment:
This is Godot's official PR pitch and not just the GitHub description, so I'd tend to keep it general and not good too deep into details. So adding "and consoles" is good, but I'd leave out the "(via third-party providers)", and possibly the link.

Reply:
I removed `(via third-party providers)` but left the link in, as the fact that console support is only available via third-party providers shouldn't be overlooked.